Gopkg.toml

```diff
@@ -90,19 +90,19 @@ required = [
 
 [[constraint]]
   name = "github.com/giantswarm/microendpoint"
-  branch = "master"
+  branch = "micrologger-0-1-0"
 
-[[constraint]]
+[[override]]
   name = "github.com/giantswarm/microerror"
-  branch = "master"
+  version = "v0.1.0"
 
 [[constraint]]
   name = "github.com/giantswarm/microkit"
-  branch = "master"
+  branch = "micrologger-0-1-0"
```

Comment:
Why use `micrologger-0-1-0` branch instead of `v0.1.0` release?

Reply:
I was testing locally and I forgot this. Anyway, the microkit release `v0.1.0` depends on micrologger `master` branch. [I tried updating it but it fails](https://github.com/giantswarm/microkit/pull/131), I believe due to a similar issue with `microerror`. I guess I'd need to do all the same PR's I did yesterday, but for `microerror`?

Reply:
Fixed with overrides

Reply:
Submitted several PR's to other projects. Working without overrides now